### ISSUE TITLE
Values with custom tags are set to `null`

### ISSUE THREAD

**Issue by GroovyCarrot** [kislyuk/yq#23]
So I have an ansible playbook which has tags like this:
```
some_key: !vault |
  $ANSIBLE_VAULT;1.1;AES256
  37663434363236326231303035313330356563343734393664373764333962663539623031346531
  6335383361386661613563386265326264613738353632310a633162393366636362633835633430
  34313038633233646432646335643336643234613133643734303530616137366132313038343035
  3039313262386130350a626364633935643562376366306631626530663037376630613565373164
  6261
```

When trying to extract this, to pipe it into a decrypt:
```
$ yq '.some_key' file.yml
null
```

It would be more useful to just ignore tags that aren't recognised and pull the value out as-is, rather than just `null`ing the value out entirely. It might be a trivial to expose tags separately if you need them?
```
$ yq --tags '.some_key' file.yml
[
  "vault"
]
```

**Comment by kislyuk** [comment]
I pushed a solution for this, but I'm wary of unexpected behavior with this. YAML tags are very troublesome and generally I'd like to do as little as possible to support them.

**Comment by kislyuk** [comment]
@GroovyCarrot and @andrewbaxter, I'd appreciate if you tested master and commented on whether this suits your needs or if you see any caveats.

**Comment by rendaw** [comment]
I like this better than the previous null stuff.  I haven't been extracting data from within the tags so it's hard for me to say if there are hidden issues with this approach.  My scripts are still working though.

**Comment by andrewbaxter** [comment]
I like this better than the previous null stuff.  I haven't been extracting data from within the tags so it's hard for me to say if there are hidden issues with this approach.  My scripts are still working though.

**Comment by kislyuk** [comment]
Released in v2.5.0.

**Comment by GroovyCarrot** [comment]
Just had a chance to test, works perfectly 👍 thanks.
